web/wwwroot/css/site.css

```diff
@@ -59,5 +59,10 @@ input.invalid {
 }
 
 .tables{

```

Comment:
Sorry I missed this earlier! Can get a space between the selector and `{` for this and the rule below please 🙂

Reply:
That's fine! I must have missed it originally or you'd have had nothing to find :')